src/Workload/Microsoft.Maui.Sdk/Sdk/Microsoft.Maui.Sdk.After.targets

```diff
@@ -1,12 +1,31 @@
 <Project>
 
-  <!-- @(ProjectCapability) used to identify parts of Maui -->
+  <!--
+    The IDE uses the @(ProjectCapability) items to activate various features.
+  -->
+
+  <!-- .NET MAUI features -->
   <ItemGroup>
     <ProjectCapability Include="Maui"           Condition=" '$(UseMaui)' == 'true' " />
     <ProjectCapability Include="MauiAssets"     Condition=" '$(UseMaui)' == 'true' or '$(UseMauiAssets)' == 'true' " />
     <ProjectCapability Include="MauiBlazor"     Condition=" '$(UsingMicrosoftNETSdkRazor)' == 'true' " />
     <ProjectCapability Include="MauiCore"       Condition=" '$(UseMaui)' == 'true' or '$(UseMauiCore)' == 'true' " />
     <ProjectCapability Include="MauiEssentials" Condition=" '$(UseMaui)' == 'true' or '$(UseMauiEssentials)' == 'true' " />
+    <!-- these 2 are potentially obsolete -->
+    <ProjectCapability Include="UseMaui"        Condition=" '$(UseMaui)' == 'true' " />
+    <ProjectCapability Include="UseMauiCore"    Condition=" '$(UseMaui)' == 'true' or '$(UseMauiCore)' == 'true' " />
```

Comment:
These need to go in `WorkloadManifest.targets`, so that Lutz can get telemetry for customers opening MAUI projects and they have no MAUI workloads installed.

Reply:
Gotcha! Good catch.